model/countries/NL/NL-model.yaml

```diff
@@ -0,0 +1,23 @@
+cut-off-children:
+  - street
+  - building
+  - unit
+
+cut-off-tokens:
+  - organization-title
```

Comment:
Let's also include  `honorific-suffix` as part of `cut-off-tokens`.

Reply:
Done, thanks!